### ISSUE TITLE
Poor alignment in calm

### ISSUE THREAD

**Issue by akashks1998** [insanum/gcalcli#372]
With `gcalcli calm 1`  I am getting the following result. How can I fix it?
It is not because of the number of events is high, I was getting the same alignment problem when I have no events. 
So, I thought it is because of my font. So, I have changed it. But then also the same problem. 
I am also getting the same problem when I was using it with` conky`.
![2018-08-31-201925_1920x1080_scrot](https://user-images.githubusercontent.com/25325900/44919584-761c5e80-ad5b-11e8-84db-df50bf708035.png)

**Comment by jcrowgey** [comment]
Hi!  I definitely refactored some of the drawing code recently, so this is probably my fault :)

I'm happy to fix it for you, but I need more information.  I can't seem to reproduce this with my own calendar using the lastest commit to master (I tried with both python27 and python3.5).

As a first step, maybe you could tell me what version of gcalcli and what version of python you're using.  As a second step, see if you can figure out how to reproduce this so that I debug it for you.

**Comment by akashks1998** [comment]
Hi, I was previously using version 3.3 of gcalcli, but then yesterday I had installed the latest version(4.0.0a5) but then also I am getting the same error.

**Comment by akashks1998** [comment]
Yesterday I had tried slack cli, in that also I was getting the same alignment problem

**Comment by jcrowgey** [comment]
Thanks!  It's helpful to know that it was an error in 3.3. and 4.0.0a5 (so it's not a regression I introduced when I refactored).  I still need repro info in order to fix this.  I don't know if you can isolate a particular event or set of events which trigger it (or locale, encoding, etc).  Without the ability to reproduce, it's really impossible to fix.

**Comment by grahamlopez** [comment]
Hello, I want to add that I see possibly similar alignment issues. When I use the default width, it does okay, but when I use the "-w" flag, some misalignment happens. This is on commit 8b2f01e with python 3.6.5.

![screenshot from 2018-09-14 14-49-40](https://user-images.githubusercontent.com/5647645/45569439-88070280-b82d-11e8-9120-17c78aa9ca39.png)

**Comment by grahamlopez** [comment]
Forgot to add, the above happens for me the same with both `calm` and `calw`.

**Comment by firecat53** [comment]
9054e06 fixes the alignment issue for me, thanks! The only outstanding issue is the today marker line seems to be not quite right (see screenshot taken at 1335 local time on 4/19). Command used:

    gcalcli --config-folder /home/firecat53/.config/gcalcli/ calm --width 16 --military

![month](https://user-images.githubusercontent.com/568113/56443206-78aad480-62a8-11e9-8556-3d17803a5d23.png)

One other unrelated note: the `calw 2` argument doesn't seem to work anymore. Haven't tried it in awhile so I don't know when it went away.

Thanks!

**Comment by jcrowgey** [comment]
@firecat53 when we switched to argparse in the 4.0.x series is when things would have changed, I believe.  In any case, the current syntax is:

```shell
$ gcalcli calw --help
usage: gcalcli calw [-h]
                    [--details {all,calendar,location,length,reminders,description,longurl,shorturl,url,attendees,email,attachments}]
                    [--color-owner COLOR_OWNER] [--color-writer COLOR_WRITER]
                    [--color-reader COLOR_READER]
                    [--color-freebusy COLOR_FREEBUSY]
                    [--color-date COLOR_DATE]
                    [--color-now-marker COLOR_NOW_MARKER]
                    [--color-border COLOR_BORDER] [--color-title COLOR_TITLE]
                    [--tsv] [--nostarted] [--nodeclined] [--width CAL_WIDTH]
                    [--military] [--override-color] [--monday] [--noweekend]
                    [start] [weeks]

Get a week-based agenda in calendar format.

positional arguments:
  start
  weeks
```

So there's two positional args and if you just put one, it's interpreted as the start arg.  If you want two weeks, this should work:

```shell
$ gcalcli calw now 2
```

**Comment by jcrowgey** [comment]
I believe the now marker was fixed in 5153f82 (v4.1.1).
